tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -826,6 +826,53 @@ def complex_agg(x, agg):
     return zip(index, res)
 
 
+@set_property("fctype", "combiner")
+def fft_aggregated(x, param):
+    """
+    Returns the spectral centroid (mean), variance, skew, and kurtosis of the absolute fourier transform spectrum.
+
+    :param x: the time series to calculate the feature of
+    :type x: pandas.Series
+    :param param: contains dictionaries {"aggtype": s} where s str and in ["centroid", "variance",
+        "skew", "kurtosis"]
+    :type param: list
+    :return: the different feature values
+    :return type: pandas.Series
+    """
+
+    assert set([config["aggtype"] for config in param]) <= set(["centroid", "variance", "skew", "kurtosis"]), \
+        'Attribute must be "centroid", "variance", "skew", "kurtosis"'
+
+
+    def get_moment(y, moment):
+        return y.dot(np.arange(len(y)) ** moment) / y.sum()
+
+    def get_centroid(y):
+        return get_moment(y, 1)
+
+    def get_variance(y):
+        return get_moment(y, 2) - get_centroid(y) ** 2
+
+    def get_skew(y):
+        return skewness(y)
+
+    def get_kurtosis(y):
+        return kurtosis(y)
+
+    calculation = dict(
```

Comment:
I like the way how you calculate the attributes by help of the calculation dictionary

Reply:
Thanks.